src/pip/_internal/wheel.py

```diff
@@ -928,9 +928,9 @@ def _base_setup_args(self, req):
         # isolating. Currently, it breaks Python in virtualenvs, because it
         # relies on site.py to find parts of the standard library outside the
         # virtualenv.
-        base_cmd = make_setuptools_shim_args(req.setup_py_path,
-                                             unbuffered_output=True)
-        return base_cmd + list(self.global_options)
+        return make_setuptools_shim_args(req.setup_py_path,
```

Comment:
This would look nicer if you put `req.setup_py_path` at the beginning of the next line (after indenting four spaces).